# Problem Set 7 - Andrew Koren

## Problem 1. Bending Magnets

a) There are 100 section that contain 8 sector magnets of length $\ell = 3.25$ m, so the total magnet length is $800\times 3.25 = 2.6\text{ km}$, yielding a fraction of $.65$.

b) $2\pi$ radians per $2.6 \text{ km}$, times $3.25$ m yields $7.86 \text{ mrad}$.

c) This comes out to $636$ m. Each bend has $\rho=\ell/\theta = 413$ m. These are different since the system isn't a continuous block of magnet and has some spacing for other elements, but there is still a total bend of $2\pi$ radians.

d) $B = {\gamma m c\beta \over q \rho} = {\gamma m c \over q \rho}\sqrt{1-{1 \over \gamma^2}}$, where $\gamma = {E \over mc^2}$

In [56]:
from scipy.constants import c, e, m_p, mu_0 # kg

mc2 = 938 # MeV
E_initial = 50e3
E_final   = 300e3

gamma_i = E_initial/mc2
gamma_f = E_final/mc2

rho = 3.25/.00786 # m

B_i = gamma_i*m_p*c/(e*rho)*(1-1/(gamma_i**2))**(1/2)
B_f = gamma_f*m_p*c/(e*rho)*(1-1/(gamma_f**2))**(1/2)
print(B_i,B_f)
print(B_f*0.1/mu_0 *1e-6)

0.40340196560778596 2.4208259899226796
0.19264321135619786


so 0.4T and 2.4T at injection and maximum, respectively

e) $\int B dl = \mu_0 N I$

$$
NI = {B l \over \mu_0} = 192643 \text{ amp-turns}
$$

<div style="page-break-before: always; text-align:right;">
  Andrew Koren
</div>

## Problem 2. Muon Storage ring

a) Combine 24.40 with 24.42 from 

$$
\langle P \rangle = mc^2 {4 \pi \over 3} {r_c \over \rho} \beta^3 \gamma^4 {I \over e}
$$

b) $\gamma = {E \over mc^2}$

$$
\begin{gathered}
    m_ec^2 = 511\text{ keV} & r_{c,e} = 2.82 \times 10^{-15} \text{ m} 
\\  m_\mu c^2 = 105\text{ MeV} & r_{c,\mu} = 1.36\times10^{-17} \text{ m}
\end{gathered}
$$
Plug into the above formula. Note that we do not divide by $e$ if we are in electron volts

In [36]:
from scipy.constants import e, pi, c
# convert to MeV
E = 30e3 # 30 GeV
m_e,  m_mu = .511, 105 # mc^2
r_ce, r_cmu= 2.82e-15, 1.35e-17 # meters
current = 1 # amps
radius = 250 # meters
gamma_e = E/m_e 
gamma_mu= E/m_mu
beta_e = (1-(1/gamma_e )**2)**(1/2)
beta_mu= (1-(1/gamma_mu)**2)**(1/2)

P_e  = m_e *(4*pi/3)*(r_ce /radius)*beta_e **3*gamma_e **4*current
P_mu = m_mu*(4*pi/3)*(r_cmu/radius)*beta_mu**3*gamma_mu**4*current
print('gamma e, mu:', gamma_e.__ceil__(), gamma_mu.__ceil__())
print('beta e, mu:', beta_e, beta_mu)
print(f'power e: {P_e:.2f}  MW')
print(f'power mu: {P_mu*1e6:.2f} W')

gamma e, mu: 58709 286
beta e, mu: 0.9999999998549327 0.999993874981242
power e: 286.83  MW
power mu: 0.16 W


In [54]:
(r_cmu * m_mu)/ (r_ce * m_e )

0.9836782279218886

c) The current will decay with time dilation $\gamma \tau_\mu$ so $286\times 2.2\times 10^{-6} = 6.3\times 10^{-4}$

d) The total muons in the beam depends on the circumference $N = {I C \over e \beta c} = {I 2\pi \rho \over e \beta c}$, the decay rate is just $\tau_\mu$

In [45]:
lifetime = gamma_mu * 2.2e-6

N_muons = current*2*pi*radius/(e*beta_mu*c)
print(f'N_muons: {N_muons:.2e}')
print(f'Heat Load: {(N_muons/lifetime)*(E*e):.0f} MW')

N_muons: 3.27e+13
Heat Load: 250 MW


<div style="page-break-before: always; text-align:right;">
  Andrew Koren
</div>


## Problem 3. RF Phase Choice

The second derivative of $V(\Delta \phi)$ w.r.t $\phi$ is

$$
\left.{\partial^2 V\over \partial \phi^2} \right|_{\phi=\phi_s} = -qE_0 T\sin(\phi_s)
$$

all answers are mod 2 pi

a) 
- Decelerating $\cos (\phi_s ) < 0 \rightarrow |\phi_s| \in \left(\pi/2, \pi\right)$, 
- focusing $-\sin(\phi_s) > 0 \rightarrow \phi_s \in \left(-\pi, 0 \right)$
- both: $\phi_s \in \left(-\pi, -\pi/2\right)$

b)

- accelerating $\cos (\phi_s) > 0 \rightarrow |\phi_s| \in \left(0, \pi/2\right)$, 
- defocusing $-\sin(\phi_s) < 0 \rightarrow \phi_s \in \left(0,\pi\right)$
- both: $\phi_s \in \left(0,\pi/2\right)$


c) This is at the maximally stable value $\theta=-\pi/2\rightarrow \left.{\partial^2 V\over \partial \phi^2} \right|_{\phi=\phi_s} = qE_0 T$ (max of the negative sine term), but that sits at the zero acceleration mark $\cos(\pi/2)=0$.

